app/store/ephemeral_store.ts

```diff
@@ -279,6 +284,21 @@ class EphemeralStoreSingleton {
     isUnacknowledgingPost = (postId: string) => {
         return this.unacknowledgingPost.has(postId);
     };
+
+    getChannelPlaybooksSynced = (serverUrl: string, channelId: string) => {
+        return this.channelPlaybooksSynced[serverUrl]?.has(channelId) ?? false;
+    };
+
+    setChannelPlaybooksSynced = (serverUrl: string, channelId: string) => {
+        if (!this.channelPlaybooksSynced[serverUrl]) {
+            this.channelPlaybooksSynced[serverUrl] = new Set();
+        }
+        this.channelPlaybooksSynced[serverUrl]?.add(channelId);
```

Comment:
Should we add a way to remove a channel if it's been in the store for quite some time?

Reply:
It shouldn't be an issue. The ephemeral store has a short live (it is in memory). For that to happen, the app should be up and running for a long time. If we ever find real issues of memory because of this, we can reconsider it, but I don't expect this to add much to the big numbers.